fdb-extensions/src/main/java/com/apple/foundationdb/clientlog/FDBClientLogEvents.java

```diff
@@ -137,12 +139,19 @@ public String getDcId() {
             return dcId;
         }
 
+        public String getTenant() {
+            return tenant;
+        }
+
         protected StringJoiner toStringBase() {
             StringJoiner joiner = new StringJoiner(", ", getClass().getSimpleName() + "[", "]")
                     .add("startTimestamp=" + getStartTimestampString());
             if (dcId.length() > 0) {
                 joiner.add("dcId=" + dcId);
             }
+            if (tenant.length() > 0) {
+                joiner.add("tenant=" + dcId);
```

Comment:
```suggestion
                joiner.add("tenant=" + tenant);
```

I assume this is what you meant